nested-spire/README.md

```diff
@@ -0,0 +1,226 @@
+
+The configuration file for the [nestedA-server](./nestedA/server/server.conf) includes the `spire` UpstreamAuthority plugin definition with the `root-server` as its upstream SPIRE Server.
+
+```console
+   UpstreamAuthority "spire" {
+ 	   plugin_data = {
+ 	       server_address      = "root-server"
+ 	       server_port         = 8081
+ 	       workload_api_socket = "/opt/spire/sockets/workload_api.sock"
+ 	   }
+    }
+```
+
+The Docker Compose definition for the `nestedA-server` service in the [docker-compose.yaml](docker-compose.yaml) file mounts the new local directory `sharedRootSocket` as a volume. Remember from the previous section that the `root-agent` socket is mounted on that directory. That way the `nestedA-server` can access the `root-agent` workload API and fetch its SVID.
+
+```console
+   nestedA-server:
+     # Share the host pid namespace so this server can be attested by the root agent
+     pid: "host"
+     image: gcr.io/spiffe-io/spire-server:0.10.1
+     hostname: nestedA-server
+     labels:
+       # label to attest nestedA-server against root-agent
+       - org.example.name=nestedA
+     volumes:
+       # Add root agent socket  
+       - ./shared/rootSocket:/opt/spire/sockets
+       - ./nestedA/server:/opt/spire/conf/server
+     command: ["-config", "/opt/spire/conf/server/server.conf"]
+```
+
+## Create Downstream Registration Entry
+
+The `nestedA-server` must be registered on the `root-server` to get SVIDs. We achieve this by creating a registration entry in the root SPIRE Server for the `nestedA-server`.
+
+```console
+   docker-compose exec -T root-server \
+       /opt/spire/bin/spire-server entry create \
+       -parentID "spiffe://example.org/spire/agent/x509pop/$(fingerprint root/agent/agent.crt.pem)" \
+       -spiffeID "spiffe://example.org/nestedA" \
+       -selector "docker:label:org.example.name:nestedA-server" \
+       -downstream \
+       -ttl 3600
+```
+
+The `-parentID` flag contains the SPIFFE ID of the `root-agent`. The SPIFFE ID of the `root-agent` is created by the [x509pop Node Attestor](https://github.com/spiffe/spire/blob/master/doc/plugin_server_nodeattestor_x509pop.md) plugin which defines the SPIFFE ID as `spiffe://<trust domain>/spire/agent/x509pop/<fingerprint>`. A `fingerprint()` function in the shell script calculates the SHA1 fingerprint of the certificate.
+The other point to highlight is the `-downstream` option. This option, when set, indicates that the entry describes a downstream SPIRE Server.
+
+## Run the Scenario
+
+Use the `set-env.sh` script to run all the services that make up the scenario. The script starts the `root`, `nestedA`, and `nestedB` services with the configuration options described earlier.
+
+Ensure that the current working directory is `.../spire-tutorials/nested-spire` and run:
```

Comment:
mmm can it be confuse to some point, in other case you used `    cd your_path/nested-spire` you must use the same in both cases

Reply:
I agree with Marcos that we should use the same form in all places. I have a slight preference for your_path/spire-tutorials/nested-spire but .../spire-tutorials/nested-spire is ok also. (Of course, use just your_path/spire-tutorials in the earlier instance to show the precise appropriate path.)

Reply:
I agree with Marcos that we should use the same form in all places. I have a slight preference for your_path/spire-tutorials/nested-spire but .../spire-tutorials/nested-spire is ok also. (Of course, use just your_path/spire-tutorials in the earlier instance to show the precise appropriate path.)